### DISCUSSION TITLE
How can I use vercel serverless function with vitesse ?

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [antfu/vitesse#264] [Q&A]
Hey I wanted to know if someone know how to use vercel function with vitesse in order to fetch database. Thanks

**Comment by thuongtruong1009** [comment]
Hi @MatteoGauthier !
![image](https://user-images.githubusercontent.com/71834167/149916825-d7515f51-9049-4b6d-9e7d-6647253e1f66.png)
- You can find more information [here](https://vercel.com/docs/concepts/functions/edge-functions)
- Hope it helps you!
- If can, you can mark it as an answer for another guy to know about this.
- Thank you!
